### PR TITLE
fix(replays): fix replay rage click option

### PR DESCRIPTION
**Pull request by JoshFerge**
needs to use `get_value_with_default` here in order to return correct value.

### CODE REVIEW

**Review by c298lee — APPROVED** [review]
do we want tests for feedback spam detection too?

**Comment by JoshFerge** [comment]
> do we want tests for feedback spam detection too?

yes, i can follow up.

File: src/sentry/api/serializers/models/project.py [hunk 1]
```diff
@@ -1046,9 +1046,11 @@ def format_options(self, attrs: dict[str, Any]) -> dict[str, Any]:
                 self.get_value_with_default(attrs, "sentry:feedback_user_report_notifications")
             ),
             "sentry:feedback_ai_spam_detection": bool(
-                options.get("sentry:feedback_ai_spam_detection")
+                self.get_value_with_default(attrs, "sentry:feedback_ai_spam_detection")
```
Comment (aliu3ntry) [inline src/sentry/api/serializers/models/project.py:1049]: Where are the defaults defined?

**Review by aliu39 — APPROVED** [review]

File: src/sentry/api/serializers/models/project.py (hunk 1 above)
Comment (JoshFerge) [reply src/sentry/api/serializers/models/project.py:1049]: https://github.com/getsentry/sentry/blob/0bb3b6292a8b9e2eb3f30a5ddf3be41e37292d57/src/sentry/projectoptions/defaults.py#L136

**Pull request merged**